src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -320,6 +320,12 @@ public async Task<IActionResult> CreatePost([Bind(Prefix = "User.Password")] str
             return View(shape);
         }
 
+        if (emailConfirmed && !await _userManager.IsEmailConfirmedAsync(user))
```

Comment:
Why do we need to do this here since we already do it in the UserDisplayDriver?

Reply:
It fails with an exception in the UserDisplayDriver, because the user doesn't exist at that stage. It's created in the controller afterwards.

Reply:
Then please should remove it from the driver. It's probably better to add this to a handler instead of adding this to the controller. This logic should be part of the registration feature not the Users feature.